lib/auth/new_web_user.go

```diff
@@ -124,7 +124,7 @@ func (s *AuthServer) GetSignupTokenData(token string) (user string, qrCode []byt
 	// It's a TOCTOU bug in the making: https://en.wikipedia.org/wiki/Time_of_check_to_time_of_use
 	_, err = s.GetPasswordHash(tokenData.User.Name)
 	if err == nil {
```

Comment:
we have `AlreadyExistsError` that I suggest to use here instead of generic `Errorf` again